src/components/IndicateBar.tsx

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { ReactComponent as Friends } from '../assets/images/Friends.svg';
+import { ReactComponent as Friends_black } from '../assets/images/Friends-black.svg';
+import { ReactComponent as Chat_black } from '../assets/images/Chat-black.svg';
+import { ReactComponent as Chat } from '../assets/images/Chat.svg';
+import { ReactComponent as User } from '../assets/images/User.svg';
+import { ReactComponent as User_black } from '../assets/images/User-black.svg';
+import { ReactComponent as Safari } from '../assets/images/Safari.svg';
+import { ReactComponent as Safari_black } from '../assets/images/Safari-Black.svg';
+import styled from 'styled-components';
+
+interface IndicateBarProps {
+    activeIcon: 'friends' | 'chat' | 'safari' | 'user';
+  }
+  
+const IndicateBar: React.FC<IndicateBarProps> = ({ activeIcon }) => {
+
+    const navigate = useNavigate();
+
+    const goToFriendList = () => {
+        navigate('/friends');
+    };
+    const goToChatList = () => {
+        navigate('/chatlist');
+    }
+    const goToUserProfile = () => {
+        navigate('/userprofile')
+    }
+    const goToGoogle = () => {
+      if (window.confirm("구글")) {
+        window.open("https://www.google.com", "_blank");
+    }
+    }
+    return (
+      <IconContainer>
+        {activeIcon === 'friends' ? 
+          <Friend_black /> : 
+          <Friend onClick={goToFriendList} />}
```

Comment:
onClick 시 실행하는 로직이 복잡하지 않으므로 따로 함수로 처리하지 않고 onClick 안에 바로 선언해줘도 좋을 것 같아요!!

```suggestion
<Friend onlick={()=>navigate(/friends)} />